Configuration/TypoScript/constants.typoscript

```diff
@@ -108,8 +108,8 @@ page {
         language {
             # cat=bootstrap package: basic/168/language_enable; type=boolean; label=LanguageSection: Enable to display the LanguageSection at bottom
             enable = 1
-            # cat=bootstrap package: basic/168/language_languageValue; type=string; label=Language Navigation Values: Comma separated list of language uids
-            languageValue = 0,1,2
+            # cat=bootstrap package: basic/168/language_languageValue; type=string; label=Language Navigation Values: Comma separated list of language uids. Leave empty to load all enabled system languages
+            languageValue = 
```

Comment:
i think we should make this more speaking, instead of providing an empty value we shoud really make clear that this is auto generated `language.languageValue = auto`

Reply:
yeah, that's a nice idea and done

Reply:
thx!